app/src/main/java/com/codesoom/assignment/exceptions/product/ProductNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment.exceptions.product;
+
+import com.codesoom.assignment.exceptions.CommonException;
+import org.springframework.http.HttpStatus;
+
+public class ProductNotFoundException extends CommonException {
+    private static final String MESSAGE = "상품이 존재하지 않습니다.";
+
+    /**
+     * 상품을 찾지 못했을 때 던집니다.
+     */
+    public ProductNotFoundException() {
```

Comment:
이 설명은 클래스에 달아주는 것이 좋겠습니다.
```suggestion
/**
 * 상품을 찾지 못했을 때 던집니다.
 */
public class ProductNotFoundException extends CommonException {
    private static final String MESSAGE = "상품이 존재하지 않습니다.";

    public ProductNotFoundException() {
```

Reply:
그러는게 좋을 것 같네요 :)
(dda7473) 수정하였습니다.